testsuite/cluster-test/src/experiments/performance_benchmark_nodes_down.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{collections::HashSet, fmt::Display};
+
+use failure::_core::fmt::{Error, Formatter};
+use failure::_core::time::Duration;
+
+use crate::cluster::Cluster;
+use crate::effects::StopContainer;
+use crate::experiments::Context;
+use crate::experiments::Experiment;
+use crate::instance::Instance;
+use crate::{effects, stats};
+
+use crate::util::unix_timestamp_now;
+
+pub struct PerformanceBenchmarkNodesDown {
+    down_instances: Vec<Instance>,
+    up_instances: Vec<Instance>,
+    percent_nodes_down: usize,
+}
+
+impl PerformanceBenchmarkNodesDown {
```

Comment:
I wonder if we can remove `PerformanceBenchmark` and just have it as `PerformanceBenchmarkNodesDown` with 0%? Would save duplication of otherwise same code?
Or maybe going further and having smth like `PerformanceBenchmark` taking list of effects to apply before running?

Just thoughts - let me know if you think thats too complex

Reply:
Removed `PerformanceBenchmark` and replaced it with `PerformanceBenchmarkNodesDown` with 0 nodes down

> Or maybe going further and having smth like PerformanceBenchmark taking list of effects to apply before running

Maybe in a future PR